config/fcos/v1_6_exp/validate_test.go

```diff
@@ -231,3 +231,129 @@ func TestValidateGrubUser(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateConfig(t *testing.T) {
+	tests := []struct {
+		config  Config
+		err     error
+		errPath path.ContextPath
+	}{
+		// valid config
+		{
+			config: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Disks: []base.Disk{
+							{
+								Device:    "/dev/vda",
+								WipeTable: util.BoolToPtr(false),
+								Partitions: []base.Partition{
+									{
+										Label:  util.StrToPtr("foo"),
+										Number: 1,
+									},
+									{
+										Label:  util.StrToPtr("bar"),
+										Number: 2,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			err:     nil,
+			errPath: path.New("yaml"),
```

Comment:
If the `err` is nil we might want to consider dropping both `err` and `errPath` ?

Reply:
didn't know you could do that, have it added now though!